core/controllers/admin_test.py

```diff
@@ -68,10 +68,10 @@
 # to PascalCase and its values to UPPER_CASE. Because we want to be consistent
 # throughout the codebase according to the coding style guide.
 # https://github.com/oppia/oppia/wiki/Coding-style-guide
-class PARAM_NAMES(enum.Enum): # pylint: disable=invalid-name
+class ParamNames(enum.Enum): # pylint: disable=invalid-name
```

Comment:
Should we remove the pylint disables?

Reply:
@nithusha21 Do I have to remove these comments or you want me to do something else?

Reply:
Yes, I believe the comments were present to bypass the longer from flagging the camel case names as incorrect style, but as we are now fixing that, the linter shouldn't be disabled anymore.

Reply:
@nithusha21  I have removed both TODO and this comment. Anything else?